### PR TITLE
[WEB3-280] EON Explorer Redesign: Copy updates

### PR DESCRIPTION
**Pull request by jmontesdeocanuez**

### CODE REVIEW

File: ui/block/BlockDetails.tsx [hunk 1]
```diff
@@ -308,8 +308,9 @@ const BlockDetails = ({ query }: Props) => {
       ) }
       { !config.UI.views.block.hiddenFields?.burnt_fees && (
         <DetailsInfoItem
-          title="Burnt fees"
+          title="Reward pool fees"
           hint={
+            //DOUBLE CHECK: Update this text
```
Comment (yurikoinaba) [inline ui/block/BlockDetails.tsx:313]: The tooltip copy can be the same as in the current explorer `ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`

File: ui/blocks/BlocksTable.tsx [hunk 2]
```diff
@@ -46,9 +46,10 @@ const BlocksTable = ({ data, isLoading, top, page, showSocketInfo, socketInfoNum
           <Th width="64px" isNumeric>Txn</Th>
           <Th width={ `${ GAS_COL_WEIGHT / widthBase * 100 }%` }>Gas used</Th>
           { !isRollup && !config.UI.views.block.hiddenFields?.total_reward &&
+              // DOUBLE CHECK: we have this other Reward column, is it ok?
               <Th width={ `${ REWARD_COL_WEIGHT / widthBase * 100 }%` }>Reward { config.chain.currency.symbol }</Th> }
           { !isRollup && !config.UI.views.block.hiddenFields?.burnt_fees &&
-              <Th width={ `${ FEES_COL_WEIGHT / widthBase * 100 }%` }>Burnt fees { config.chain.currency.symbol }</Th> }
+              <Th width={ `${ FEES_COL_WEIGHT / widthBase * 100 }%` }>Reward pool fees { config.chain.currency.symbol }</Th> }
```
Comment (yurikoinaba) [inline ui/blocks/BlocksTable.tsx:52]: What is the difference, i.e. is the Reward column empty?

I'd like to compare the data displayed in the redesign w the current explorer and make sure they are the same for the Reward pool fees colum.

File: ui/blocks/BlocksTableItem.tsx [hunk 3]
```diff
@@ -119,7 +119,7 @@ const BlocksTableItem = ({ data, isLoading, enableTimeIncrement }: Props) => {
               { burntFees.dividedBy(WEI).toFixed(8) }
             </Skeleton>
           </Flex>
-          <Tooltip label={ isLoading ? undefined : 'Burnt fees / Txn fees * 100%' }>
+          <Tooltip label={ isLoading ? undefined : 'Reward pool fees / Txn fees * 100%' }>
```
Comment (yurikoinaba) [inline ui/blocks/BlocksTableItem.tsx:122]: The same tooltip copy can be used here too `ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`

File: ui/tx/TxDetails.tsx [hunk 4]
```diff
@@ -431,7 +431,8 @@ const TxDetails = () => {
         ) }
         { data.tx_burnt_fee && !config.UI.views.tx.hiddenFields?.burnt_fees && !config.features.optimisticRollup.isEnabled && (
           <DetailsInfoItem
-            title="Burnt fees"
+            title="Reward pool fees"
+            // DOUBLE CHECK: is this calculation correct with the new naming?
```
Comment (yurikoinaba) [inline ui/tx/TxDetails.tsx:435]: `ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`

File: ui/blocks/BlocksTable.tsx (hunk 2 above)
Comment (jmontesdeocanuez) [reply ui/blocks/BlocksTable.tsx:52]: I haven't seen any example of this, but after doing some reading, my understanding is that in some blockchain systems, fees are typically divided into two parts: rewards (portion of the fees allocated as rewards to validators/miners) and burnt fees (portion of the fees that are removed from circulation), and those are the parts here, that tipically are different but in our case we use the burnt fees as rewards.

I'm not sure why we use burnt fees as rewards, if the system has a dedicated field for them, but as we're considering burnt fees as rewards, I think we should be good to remove the reward column.

What do you think @yurikoinaba ?

**Comment by jmontesdeocanuez** [comment]
> * Do you want a development branch?
I'd say we will needed for the pipeline when we are ready for release this, so we can have a dev environment (pointing to development branch) and a prod environment (pointing to main). However, while we continue developing v1, I think is not strictly needed.

> * The requirement to merge to main branch can be made to 1 approval if you'd like
That'd be good for now to accelerate the development process.

**Review by yurikoinaba — APPROVED** [review]

File: types/api/reward.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
 export interface Reward {
   reward: string;
-  type: 'Miner Reward' | 'Validator Reward' | 'Emission Reward' | 'Chore Reward' | 'Uncle Reward' | 'POA Mania Reward';
+  type: 'Miner Reward' | 'Validator Reward' | 'Emission Reward' | 'Chore Reward' | 'POA Mania Reward';
```
Comment (davehorizen) [inline types/api/reward.ts:3]: Probably it is outside the scope of this PR but i think we can also remove the POA reward as it part of the Gnosis ecosytem only. Don't know it there are reference to this on the current front end but if so I will remove it

**Review by davehorizen — APPROVED** [review]
lgtm, I left a comment regarding a reward that can be outside the scope of our sidechain

File: types/api/reward.ts (hunk 5 above)
Comment (jmontesdeocanuez) [reply types/api/reward.ts:3]: Thanks Davide, I'll make a note of it and keep it in mind if we ever see any references to POA in our project. With the current env variables, it shouldn't, but definitely something to take a look on. Thank you!

**Pull request merged**